components/automate-cli/pkg/verifyserver/utils/awsutils/aws.go

```diff
@@ -0,0 +1,74 @@
+package awsutils
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+type AwsUtilsImpl struct {
+}
+
+type AwsUtils interface {
+	NewSessionWithOptions(endpoint, accessKey, secretKey, region string) (*session.Session, error)
+	DeleteObject(s3Client *s3.S3, BucketName, BasePath string) (*s3.DeleteObjectOutput, error)
+	ListObjectsV2(s3Client *s3.S3, BucketName, BasePath string) (*s3.ListObjectsV2Output, error)
+	ListBuckets(s3Client *s3.S3) (*s3.ListBucketsOutput, error)
+	NewUploader(sess *session.Session, BucketName, BasePath string) (*s3manager.UploadOutput, error)
+	New(sess *session.Session) *s3.S3
+}
+
+func NewAwsUtils() AwsUtils {
+	return &AwsUtilsImpl{}
+}
+
+func (au *AwsUtilsImpl) NewSessionWithOptions(endpoint, accessKey, secretKey, region string) (*session.Session, error) {
+	var config aws.Config
+	if region == "" {
+		region = "us-east-1"
```

Comment:
if region id empty (config.toml), and bucket is in some other region other than us-east-1, with it work or throws error ?

Reply:
if you give region as empty, i am assigning Default region **us-east-1**, if bucket is in another region its mandatory to give

Reply:
we don’t need region in minio, so in this case we need to assign default region to get aws session